omero/sysadmins/mail.txt

```diff
@@ -0,0 +1,88 @@
+OMERO.mail
+==========
+
+The OMERO server has the ability to send email to any users who have a properly
+configured email address. OMERO system administrators can then use the
+:omerocmd:`admin email` command to contact those users.
+
+In order to activate the subsystem, minimally the :property:`omero.mail.config`
+property will need to be activated.
+
+Minimum configuration
+---------------------
+
+::
+
+    $ bin/omero config set omero.mail.config true
+
+By default, this will use "localhost" as the mail server on port 25 and send as
+the user "omero". Likely you will need to configure OMERO to use your actual mail server.
+
+For example,
+
+::
+
+    $ bin/omero config set omero.mail.host smtp.university.example
+
+If authentication is required, then also configure:
+
+::
+
+    $ bin/omero config set omero.mail.username USER
+    $ bin/omero config set omero.mail.password PASS
+
+Setting email addresses
+-----------------------
+
+For the any user to receive email, a valid email address must be configured. By
+default, the `root` OMERO user will *not* have an email address configured. This
+can be done from one of the UIs or via the :omerocmd:`obj` command:
+
+::
+
+    $ bin/omero obj update Experimenter:0 email=root@university.example
+
+.. note:: Using a mailing list or an alias for the `root` user can simplify configuration.
+
+Enabling mail notifications
+---------------------------
+
+A number of "mail senders" are available for sending notifications of certain
+events on the server. Those available include:
+
+* `ServerUpMailSender` and `ServerDownMailSender` which mail when the server goes up or down
+* `FailedLoginMailSender` which can be configured to send for particular users if a bad password is used
+* `ObjectMailSender` which can be configured to send an email under various conditions. Instances which are configured include:
+   * `newUserMailSender` which sends an email every time a user is created
+   * `newCommentMailSender` which sends an email every time a user's image is commented on by another user.
+
+To activate the senders, the :file:`etc/blitz/mail-senders.example` can be copied
+to a file ending with ".xml".
```

Comment:
Copied to `etc/blitz/mail-senders.xml`. Server down email worked, Login failure results in a Blitz error:
```
2015-03-18 09:40:48,288 ERROR [  ome.security.auth.JdbcPasswordProvider] (.Server-13) LoginAttemptMessage error
java.lang.NullPointerException: null
        at ome.services.mail.FailedLoginMailSender.onApplicationEvent(FailedLoginMailSender.java:75) ~[server.jar:na]
```